src/server.js

```diff
@@ -70,7 +70,7 @@ function createRemixRequest(request, options = {}) {
     headers: request.headers,
     signal: controller.signal,
     // eslint-disable-next-line unicorn/no-null -- Request init expects a `null` value.
-    body: isGetOrHead(request) ? request.body : null,
+    body: isGetOrHead(request) ? null : request.clone().body,
   };
```

Comment:
Do we want to leave the `.clone()` here?

Reply:
You don't really need a `clone` in this context as a new request is being created below.

Reply:
Wouldn't that be an issue within the middleware provider?